three/three.def

```diff
@@ -1,6 +1,6 @@
-LIBRARY datadog-agent-two
+LIBRARY datadog-agent-three
```

Comment:
I think we can get rid of the `def` files altogether; I don't think we're using them any more. But that can be a separate PR; it's not restricted to this change

Reply:
Will do, thanks for confirming I suspected something like this since the typo seemed too big to get unnoticed